packages/now-cli/src/util/ignored.ts

```diff
@@ -14,8 +14,8 @@ export default `.hg
 .DS_Store
 .wafpicke-*
 .lock-wscript
-.env
-.env.*
+.env.local
+.env.*.local
```

Comment:
This also needs to be updated in `now-client.

Reply:
Updated